ignite/contrib/metrics/ExpectedCalibrationError.py

```diff
@@ -0,0 +1,57 @@
+import torch
```

Comment:
First, let's put it into `ignite/metrics/ExpectedCalibrationError.py` instead of `ignite/contrib/metrics/ExpectedCalibrationError.py`